tests/e2e.test.js

```diff
@@ -49,49 +49,53 @@ beforeAll(() => {
 describe.each([
   ['desktop', DEFAULT_DEVICE],
   ['mobile', devices['Nexus 7']],
-])('General (%s)', (name, device) => {
-  beforeEach(async () => {
-    await page.emulate(device);
-    await page.goto(url.href);
-  });
+])(
+  'General (%s)',
+  (name, device) => {
+    beforeEach(async () => {
+      await page.emulate(device);
+      await page.goto(url.href);
+    });
 
-  it('produces a screenshot', async () => {
-    await page.screenshot({
-      path: path.resolve(ARTIFACTS_DIR, `${name}.png`),
+    it('produces a screenshot', async () => {
+      await page.screenshot({
+        path: path.resolve(ARTIFACTS_DIR, `${name}.png`),
+      });
     });
-  });
 
-  it('displays the text "Simple Icons"', async () => {
-    await expect(page).toMatch('Simple Icons');
-  });
+    it('displays the text "Simple Icons"', async () => {
+      await expect(page).toMatch('Simple Icons');
+    });
 
-  it('has the search input in view on load', async () => {
-    const $searchInput = await page.$('#search-input');
-    expect(await isInViewport($searchInput)).toBeTruthy();
-  });
+    it('has the search input in view on load', async () => {
+      const $searchInput = await page.$('#search-input');
+      expect(await isInViewport($searchInput)).toBeTruthy();
+    });
 
-  it('has the order controls in view on load', async () => {
-    const $orderAlphabetically = await page.$('#order-alpha');
-    expect(await isInViewport($orderAlphabetically)).toBeTruthy();
+    it('has the order controls in view on load', async () => {
+      const $orderAlphabetically = await page.$('#order-alpha');
+      expect(await isInViewport($orderAlphabetically)).toBeTruthy();
 
-    const $orderByColor = await page.$('#order-color');
-    expect(await isInViewport($orderByColor)).toBeTruthy();
-  });
+      const $orderByColor = await page.$('#order-color');
+      expect(await isInViewport($orderByColor)).toBeTruthy();
+    });
 
-  it('has the first grid item in view on load', async () => {
-    const $firstGridItem = await page.$('.grid-item');
-    expect(await isInViewport($firstGridItem)).toBeTruthy();
-  });
+    it('has the first grid item in view on load', async () => {
+      const $firstGridItem = await page.$('.grid-item');
+      expect(await isInViewport($firstGridItem)).toBeTruthy();
+    });
 
-  it('hides the #copy-input element', async () => {
-    const $copyInput = await page.$('#copy-input');
-    expect(await isHidden($copyInput)).toBeTruthy();
-  });
+    it('hides the #copy-input element', async () => {
+      const $copyInput = await page.$('#copy-input');
+      expect(await isHidden($copyInput)).toBeTruthy();
+    });
 
-  afterAll(async () => {
-    await page.emulate(DEFAULT_DEVICE);
-  });
-});
+    afterAll(async () => {
+      await page.emulate(DEFAULT_DEVICE);
+    });
+  },
+  100000,
```

Comment:
This could freeze tests and spend all the simple-icons monthly Github Actions time. To prevent it, also, don't make PRs from branches of the same repo, do them always from your fork.

Reply:
this is runaway code, not an actual change. I will remove it.